source/main.ts

```diff
@@ -68,6 +69,12 @@ global.log = {
  */
 let zettlr: Zettlr|null = null
 
+/**
+ * Tray
+ * @type {Tray|null}
+ */
+let tray: Tray|null = null
+
```

Comment:
#6 is about adding the events (`'click'`) code. But it is hard to add the 'click' code when the tray isn't ready. The tray is in-progress (see #15 and #37).
As per #37 the tray code should be `source/main/modules/window-manager/index.ts` (not `source/main.ts`)